saleor/graphql/product/bulk_mutations/products.py

```diff
@@ -173,12 +173,19 @@ class BulkAttributeValueInput(InputObjectType):
     id = graphene.ID(description="ID of the selected attribute.")
     values = graphene.List(
         graphene.NonNull(graphene.String),
-        required=True,
+        required=False,
         description=(
             "The value or slug of an attribute to resolve. "
             "If the passed value is non-existent, it will be created."
         ),
     )
+    boolean = graphene.Boolean(
+        required=False,
+        description=(
+            "The boolean value of an attribute to resolve."
+            "If the passed value is non-existent, it will be created."
+        ),
+    )
```

Comment:
I think those changes are not related to Date & DateTime attributes. We should add them in separate PR. 
If we add changes not related to PR we can easily miss that we should add more tests for this bugfix. Our QA can miss that changes and don't perform required tests.

Reply:
QA is aware of the addition of the variant creator for boolean values here. 
It was explicitly said it will be added here since Paweł was out while handling boolean attrs.